src/sentry/api/endpoints/organization_integration_repos.py

```diff
@@ -61,6 +67,7 @@ def get(
                     defaultBranch=repo.get("default_branch"),
                 )
                 for repo in repositories
+                if repo["identifier"] not in external_names
```

Comment:
i'm not a fan of using the `identifier` (e.g. `getsentry/sentry`) but `install.get_repositories()` doesn't return repo ids https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/github/integration.py#L168-L175

```python
return [
    {
        "name": i["name"],
        "identifier": i["full_name"],
        "default_branch": i.get("default_branch"),
    }
    for i in response.get("items", [])
]

Reply:
this is also a behavior of the superclass (not returning the `id`)

Reply:
should we make a ticket to fix this or do sumn about it?

Reply:
the change would be small but it would require touching other integrations since we would be expecting all other integrations to also return `id` in the response for this function. i'm not sure if the other APIs always provide the `id` when we call them though cc @AniketDas-Tekky